components/__tests__/Checkbox-test.js

```diff
@@ -70,23 +70,23 @@ describe('Checkbox', () => {
       <Checkbox id="test" disabled />
     );
 
-    const input = wrapper.find('input');
-    expect(input.props().disabled).toEqual(true);
+    const input = () => wrapper.find('input');
```

Comment:
It does seem like DRYing this out with a helper function might be worth it, given that it's being used in multiple places.

Something like:

```js
const getInput = wrapper => wrapper.find('input');
```

And then:
```js
expect(getInput(wrapper).props().defaultChecked).toBeUndefined();
```